### ISSUE TITLE
4.x.x drivers are not working properly with unicode column names

### ISSUE THREAD

**Issue by pejold** [Microsoft/msphpsql#138]
4.x.x Driver: sqlsrv_field_metadata returns column name as char*, so unicode column names can't be fetched correctly

**Comment by meet-bhagdev** [comment]
@pejold We need more information to further triage this. Are you using the driver on Windows/Linux? What version of PHP are you using? Also it would be great for you to send us a repro script/schema we can use to repro this bug?

**Comment by david-garcia-garcia** [comment]
See sample script for this issue here:

http://pastebin.com/XKpd0vpr

(Make sure that script source is in UTF **WITHOUT BOM** before running it otherwise the PDO driver will complain about some utf-16 thing).

Unicode column names are all messed up when they arrive to PHP.

**Current script output:**

```
OK
FAIL
FAIL
id
v�riable
t�sting
```

**Expected output**

```
OK
OK
OK
id
väriable
tésting
```

**Comment by bdaub** [comment]
Same issue here.
I use an existing DB on our SQL-Server in Laravel 5.  Many tables and columns contain german umlauts. I've set the charset of the connection to "utf8" and can access tables like "Läger" without problems. But I couldn't access columns with umlauts. Every query returns something like

`["Stra�e"]=> string(15) "Mainzer Str. 97" `
instead of
`["Straße"]=> string(15) "Mainzer Str. 97" `

The returned values in the columns are displayed correctly.
I've tested different versions of the driver and different versions of PHP.

**Comment by bdaub** [comment]
@meet-bhagdev Any idea how to fix this issue?

**Comment by l736k** [comment]
@meet-bhagdev i'm looking for a solution too

**Comment by meet-bhagdev** [comment]
@l736k, @bdaub @david-garcia-garcia @pejold  We are fixing this issue right now and should have something to share in the coming weeks.

**Comment by david-garcia-garcia** [comment]
Confirmed to be fixed in the 4.1.5 windows release. Just note that the repo script was missing the following to work as expected:
